### PR TITLE
Events page edits - Date fixes

### PR DESCRIPTION
**Pull request by krisfreedain**
Signed-off-by: Kris Freedain <freedak@amazon.com>

### Description
Had to update a few dates and UTC times with the current pages

### Check List
- [x ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _events/2022-0124.markdown [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+eventdate: 2022-01-24T09:00
+tz: UTC -8
+title: OpenSearch Community Meeting - 2022-01-24 - open office hour
 online: true
 signup:
-    url: https://www.meetup.com/OpenSearch/events/282613028/
+    url: https://www.meetup.com/OpenSearch/events/282613041/
     title: Join on Meetup
 
 ---
```
Comment (stockholmux) [inline _events/2022-0124.markdown:12]: Maybe we should include that has been moved to accommodate MLK?

**Review by stockholmux — CHANGES_REQUESTED** [review]
One small change.

File: _events/2022-0124.markdown (hunk 1 above)
Comment (krisfreedain) [reply _events/2022-0124.markdown:12]: Edit complete

**Review by stockholmux — APPROVED** [review]
LGTM

**Comment by krisfreedain** [comment]
Thank you

**Pull request merged**